doc/tutorials/vuart_configuration.rst

```diff
@@ -334,13 +334,12 @@ The ACRN vUART related XML fields:
 
 Run the command to build ACRN with this XML configuration file::
 
-  make BOARD_FILE=$PWD/misc/acrn-config/xmls/board-xmls/<board>.xml \
-     SCENARIO_FILE=$PWD/misc/acrn-config/xmls/config-xmls/<board>/<scenario>.xml
+  make BOARD=<board> SCENARIO=<scenario>
 
 The configuration tools will test your settings, and check :ref:`vUART
 Rules <index-of-vuart>` for compilation issue. After compiling, you can find
-``./misc/vm_configs/scenarios/<scenario>/<board>/pci_dev.c`` has been
-changed by the configuration tools based on the XML settings, something like:
+the generated sources under ``build/configs/<scenario>/<board>/pci_dev.c``
```

Comment:
```suggestion
the generated sources under ``build/configs/scenarios/<scenario>/pci_dev.c``
```

There is also an instance of `misc/vm_configs/scenarios/<xxx>/vm_configuration.c` in this file (above) that should be updated.